uv.gyp

```diff
@@ -419,6 +419,11 @@
             'test/runner-unix.h',
           ],
         }],
+        [ 'OS in "mac dragonflybsd freebsd linux netbsd openbsd".split()', {
```

Comment:
Just copied that from https://github.com/Gottox/libuv/blob/check_before_reopen_tty/uv.gyp#L259